### DISCUSSION TITLE
v2.0.2 discord rpc button "View app on github"

### DISCUSSION THREAD

**Discussion by vensoran-studios** [pear-devs/pear-desktop#1288] [General]
just thinking is there a way to remove this button on discord rpc?

**Comment by JellyBrick** [comment]
https://github.com/th-ch/youtube-music/commit/575a643e55f7fe5bd89fc432ca29abbf596318ab 
This change appears to have been added by this commit.
I'll make this change optional, allowing users to show or hide it.
